man/adoc/bpftrace.adoc

```diff
@@ -3852,6 +3852,12 @@ Default: 1
 
 Controls whether maps are printed on exit. Set to `0` in order to change the default behavior and not automatically print maps at program exit.
 
+==== no_empty_posarg_warnings
```

Comment:
As mentioned, we don't need a new config flag for this. Users can just use `--no-warnings`. I'm also thinking about adding a general config flag for suppressing warnings.